hr/hr_ajout_user.php

```diff
@@ -203,17 +202,15 @@ function insertSoldeUtilisateur(array $data, \includes\SQL $sql) : bool
     $req = "INSERT INTO conges_solde_user (su_id, su_login, su_abs_id, su_nb_an, su_solde, su_reliquat) VALUES " . implode(",", $valuesStd);
     $returnStd = $sql->query($req);
     $returnExc = 1;
-    if ($config->isCongesExceptionnelsActive()) {
-        $typeAbsencesExceptionnels = \App\ProtoControllers\Conge::getTypesAbsences($sql, 'conges_exceptionnels');
-        foreach ($typeAbsencesExceptionnels as $typeId => $info) {
-            $valuesExc[] = "(DEFAULT, '" . $data['login'] . "' ,"
-                                . $typeId . ", 0, "
-                                . $data['soldes'][$typeId] . ", 0)" ;
+    $typeAbsencesExceptionnels = \App\ProtoControllers\Conge::getTypesAbsences($sql, 'conges_exceptionnels');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\App\ProtoControllers\Conge' in method 'insertSoldeUtilisateur'.](https://app.codacy.com/manual/Libertempo/web/pullRequest?prid=5122180)